### PR TITLE
Get posts method

### PR DESCRIPTION
**Pull request by dholms**
Add a method for getting an array of post views

Added to both PDS & AppView

### CODE REVIEW

File: packages/bsky/tests/views/posts.test.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+    const authed = await agent.api.app.bsky.feed.getPosts(
+      { uris },
+      { headers: sc.getHeaders(sc.dids.alice, true) },
+    )
+    const unauthed = await agent.api.app.bsky.feed.getPosts({
+      uris,
+    })
+    const stripped = authed.data.posts.map((p) => stripViewerFromPost(p))
+    // @TODO uncomment
```
Comment (dholms) [inline packages/bsky/tests/views/posts.test.ts:63]: I could use an eye on this @devinivy 

I fought with it for like 30 min & feel like I'm missing something obvious

Logging the stripped data looks totally fine, but for some reason in the jest compare, it says that `viewer` is still around 🧐

Comment (devinivy) [reply packages/bsky/tests/views/posts.test.ts]: Ah, it was just an issue with `stripViewerFromPost()` not dealing with record-with-media embeds.  Just pushed a fix to this branch 👍

**Review by devinivy — APPROVED** [review]

File: packages/bsky/tests/views/posts.test.ts (hunk 1 above)
Comment (dholms) [reply packages/bsky/tests/views/posts.test.ts]: well now i feel silly. why tf it was it console logging it out just fine then?

Comment (dholms) [reply packages/bsky/tests/views/posts.test.ts]: ah well 🤷‍♂️

Comment (devinivy) [reply packages/bsky/tests/views/posts.test.ts]: Ah, it's all good!  No idea, sounds like a ghost in the machine.  I def spent a little time looking through `unauthed` when the issue was in `stripped`.

**Pull request merged**